vlanBridge.go

```diff
@@ -436,3 +442,36 @@ func (vl *VlanBridge) processArp(pkt protocol.Ethernet, inPort uint32) {
 		}
 	}
 }
+
+// SendGARP sends GARP for the specified IP, MAC
+func (vl *VlanBridge) SendGARP(ip net.IP, mac net.HardwareAddr, vlanID uint16) error {
+	garpPkt, _ := protocol.NewARP(protocol.Type_Request)
```

Comment:
@jojimt: The idea is that this API would be used by netplugin to send out GARPs when a container is launched. The datapath is chosen depending on the network the container is launched in.